File: ProjectFindWEB/ProjectFind/src/app/models/projects.ts
export class Projects {
    id: number;
    name: string;
    description: string;
    user_Id: number;
    user_Name: string;
    user_Email: string;
    constructor(
        id?: number,
        name?: string,
        description?: string,
        user_Id?: number,
        user_Name?: string,
        user_Email?: string

    ) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.user_Id = user_Id;
        this.user_Name = user_Name;
        this.user_Email = user_Email;
    }
}

File: ProjectFindDatabase/Models/Projects.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProjectFindDatabase.Models
{
    public class Projects
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int User_Id { get; set; }
        public string User_Name { get; set; }
        public string User_Email { get; set; }
        public bool Closed { get; set; }
        public string Company_Id { get; set; }
    }
}
